spec/classes/postfix_spec.rb

```diff
@@ -224,71 +224,71 @@
           context 'when specifying a custom mastercf_content' do
             let(:params) do
               {
-                mastercf_content: 'testy',
+                'mastercf_content' => 'testy',
```

Comment:
Was this broken?

Reply:
> I've left a few comments. I think some changes are out of the scope of your change (if required at all).

Yeah. Initial change was only manifests/config.pp update and spec tests for it.
However overall test were failing so Alex assisted with fixing them and I've merged his changes.
After that #291 was merged with fixes, so we might need to review all that was done and resolve merge conflicts.

Reply:
The `key: 'value'` vs `'key' => 'value'` is linked to Ruby 1.8 vs 1.9+. There really shouldn't be a need to revert to the 1.8 syntax.